ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-bgp.j2

```diff
@@ -56,6 +56,9 @@ router bgp {{ router_bgp.as }}
 {%             if router_bgp.neighbors[neighbor].remote_as is defined and router_bgp.neighbors[neighbor].remote_as is not none %}
    neighbor {{ neighbor }} remote-as {{ router_bgp.neighbors[neighbor].remote_as }}
 {%             endif %}
+{%             if router_bgp.neighbors[neighbor].next_hop_self is defined and router_bgp.neighbors[neighbor].next_hop_self is not none %}
```

Comment:
Please change to : {%             if router_bgp.neighbors[neighbor].next_hop_self is defined and router_bgp.neighbors[neighbor].next_hop_self == true %}

Reply:
Fixed